src/alfasim_sdk/default_tasks.py

```diff
@@ -168,6 +168,10 @@ def compile(ctx, cmake_extra_args="", debug=False, clean=False):
         print_message(f"cmake args = {cmake_args}", color=Fore.YELLOW, bright=True)
 
     cmake_cmd = shutil.which("cmake")
+    assert cmake_cmd is not None, (
+        "Cmake is not available. Please install at least version 3.5.2 to compile the plugin"
+    )
+
```

Comment:
`assert` is meant for internal sanity checks, but this is a runtime error.



```suggestion
    raise invoke.Exit(message="Cmake is not available. Please install at least version 3.5.2 to compile the plugin"
    )

```

Reply:
Done!

Reply:
@nicoddemus can I close this conversation since you've apporved the PR?

Reply:
Yes, whenever someone leaves comments but approves, feel free to close the conversation later.